lib/api_remote.ml

```diff
@@ -252,11 +252,22 @@ module Buildkite : Api.Buildkite = struct
     | None -> Lwt.return @@ fmt_error "%s: failed to retrieve Buildkite access token" name
     | Some access_token ->
       let headers = bearer_token_header access_token :: Option.default [] headers in
-      let url = sprintf "https://api.buildkite.com/v2/%s" path in
+      let url =
+        if String.starts_with ~prefix:"https://api.buildkite.com/v2/" path then path
+        else sprintf "https://api.buildkite.com/v2/%s" path
+      in
       (match%lwt buildkite_api_request ?body ~headers meth url read with
       | Ok res -> Lwt.return @@ Ok res
       | Error e -> Lwt.return @@ fmt_error "%s: failure : %s" name e)
 
+  let get_job_log ~ctx (job : Buildkite_t.job) =
+    match job.log_url with
+    | None -> Lwt.return_error "Unable to get job log, job has no log_url field"
+    | Some log_url ->
+      (match%lwt request_token_auth ~name:"get buildkite job logs" ~ctx `GET log_url Buildkite_j.job_log_of_string with
+      | Ok logs -> Lwt.return_ok logs
+      | Error e -> Lwt.return_error e)
```

Comment:
```suggestion
     request_token_auth ~name:"get buildkite job logs" ~ctx `GET log_url Buildkite_j.job_log_of_string
```

no point matching here as nothing is being done

Reply:
done